sklearn/ensemble/_hist_gradient_boosting/tests/test_splitting.py

```diff
@@ -9,8 +9,13 @@
 from sklearn.utils._testing import skip_if_32bit
 
 
+@pytest.fixture
+def n_threads():
+    return 1
```

Comment:
I think it's more idiomatic to define a `N_THREADS` constant rather than pass a fixture to every single test. Fixtures are useful when there's a minimum of logic or when we need to parametrize something. Here's it's really just a constant